.markdownlint-cli2.cjs

```diff
@@ -0,0 +1,30 @@
+const githubMarkdownOpinions = require('@github/markdownlint-github')
+
+// Rules we want to turn on but currently have too many violations
+const rulesToEnforce = {
+  'fenced-code-language': false,
+  'no-duplicate-header': false, // Fix https://github.com/primer/doctocat/issues/527, then set this rule to `siblings_only: true`
```

Comment:
@primer/react-reviewers FYI,`no-duplicate-header` should be turned on because doctocat messes up the anchor links when headings have the same text. See linked issue. I keep it off for now because there's too many address.

Ideally, the issue is fixed in doctocat so we don't have to set this rule to be strict.